Source/Core/Common/FileUtil.cpp

```diff
@@ -61,7 +61,7 @@ static std::string s_android_sys_directory;
 #ifdef _WIN32
 FileInfo::FileInfo(const std::string& path)
 {
-  m_exists = _tstat64(UTF8ToTStr(path).c_str(), &m_stat) == 0;
+  m_exists = PathFileExistsW(UTF8ToUTF16(path).c_str());
```

Comment:
Doesn't this break all functions in `FileInfo` other than `Exists`?

Reply:
Yeah, I still need to keep stat around >.<